momepy/functional/_elements.py

```diff
@@ -295,3 +296,44 @@ def verify_tessellation(tesselation, geometry):
             stacklevel=2,
         )
     return collapsed, multipolygons
+
+
+def get_nearest_street(
+    buildings: gpd.GeoSeries | gpd.GeoDataFrame,
```

Comment:
Is there is specific reasoning to using both `pd` and `Series` above, and using `gpd.GeoSeries` here? Should we also do `from gpd import GeoSeries` to conform? Or vice-versa?

Reply:
No particular reason. My thought was that people will be familiar with Series but may not be with GeoSeries, so I wanted to indicate where it comes from. Do you have any preference here?

Reply:
My personal preference is "always explicit" so if I'd had my way it would be non-aliased & full usage: e.g.:

```python
import pandas
...
pandas.Series
```

However, I know that my preference is not norm. If I had to choose one of the two methods already being used here, I'd probably go with non-aliased classes (e.g. `from gpd import GeoSeries`) for the type hinting.

Reply:
I also know that you like linting rules, so have one for you - https://docs.astral.sh/ruff/rules/unconventional-import-alias/